op-challenger/challenger.go

```diff
@@ -19,5 +19,6 @@ func Main(ctx context.Context, logger log.Logger, cfg *config.Config) error {
 		return fmt.Errorf("failed to create the fault service: %w", err)
 	}
 
-	return service.MonitorGame(ctx)
+	service.MonitorGame(ctx)
+	return nil
```

Comment:
Isn't this going to just result in the program exiting immediately? The main thread isn't blocking anymore.

Reply:
Also, why do we no longer return an error in service.MonitorGame?